src/pages/api/v1/youfocus/index.ts

```diff
@@ -0,0 +1,46 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { connectDB } from '@/middlewares';
+import { apiStatusCodes } from '@/constant';
+import { addPlaylist } from '@/utils';
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  await connectDB();
+
+  switch (req.method) {
+    case 'POST':
+      return handleAddPlaylist(req, res);
+    default:
+      return res.status(apiStatusCodes.BAD_REQUEST).json({
+        success: false,
+        message: `Method ${req.method} not allowed`,
+      });
+  }
+};
+
+const handleAddPlaylist = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const { playlistUrl }: { playlistUrl: string } = req.body;
+
+    if (!playlistUrl) {
+      return res.status(apiStatusCodes.BAD_REQUEST).json({
+        success: false,
+        message: 'Playlist URL is required',
+      });
+    }
+
+    const message = await addPlaylist(playlistUrl);
```

Comment:
Read how other APIs are created. We get data and error in return.

Use the same code style.